fl4health/clients/evaluate_client.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         device: torch.device,
         loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
         model_checkpoint_path: Optional[Path] = None,
+        seed: Optional[int] = None,
```

Comment:
since evaluate_client doesn't call super, you just need to set the seed manually here. I.e. just
```
self._maybe_fix_random_seeds(seed)
```
in the at the beginning of the __init__